### PR TITLE
add notes to `gcloud container clusters`

### PR DESCRIPTION
**Pull request by zeroasterisk**
I ran into problems with not having a default zone or region.

NOTE: I was not able to get it to work specifying a region on CLI, instead I had to specify a zone. (bug?)

### CODE REVIEW

File: tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md [hunk 1]
```diff
@@ -295,8 +295,12 @@ These are clusters of VMs in the cloud, managed by a Kubernetes server.
     It might take several minutes for the cluster to be created. You can check
     the cloud console at http://cloud.google.com/console, under the Kubernetes
     Engine section, to see that your cluster is running. You will also be able
-    to see the individual running VMs under the Compute Engine section. Note
-    that once the cluster is running, you will be charged for the VM usage.
+    to see the individual running VMs under the Compute Engine section.
+    
+    Note that once the cluster is running, *you will be charged for the VM usage*.
+    
+    If you get an error "v1 API cannot be used to access GKE regional clusters" or are told to specify a zone or region,
```
[the remark this answers is not in the document]
Comment (zeroasterisk) [reply tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: Hmm... Ok. Let me clear all my config and defaults and I'll confirm.

[the remark this answers is not in the document]
Comment (jmdobry) [reply tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: Has this been confirmed?

[the remark this answers is not in the document]
Comment (zeroasterisk) [reply tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: Just updated my gcloud cli sdk.

With no default zone/region, asks for one
```
$ gcloud compute project-info describe --project $PROJECT_ID | ag zone

$ gcloud container clusters create hello-cluster --num-nodes=2
ERROR: (gcloud.container.clusters.create) One of [--zone, --region] must be supplied: Please specify location..
```

Add a default, doesn't respect it
```
$ gcloud compute project-info add-metadata \
    --metadata google-compute-default-region=us-central1,google-compute-default-zone=us-central1-a

$ gcloud compute project-info describe --project $PROJECT_ID | ag zone
  - key: google-compute-default-zone

$ gcloud container clusters create hello-cluster --num-nodes=2
ERROR: (gcloud.container.clusters.create) One of [--zone, --region] must be supplied: Please specify location..
```

Add a region flag, gets this error
```
$ gcloud container clusters create hello-cluster --num-nodes=2 --region=us-central1
ERROR: (gcloud.container.clusters.create) ResponseError: code=400, message=v1 API cannot be used to access GKE regional clusters. See https://goo.gl/uHKp3k for more information.
```

Add a zone flag, works fine
```
$ gcloud container clusters create hello-cluster --num-nodes=2 --zone=us-central1-a
(started fine)
```

[the remark this answers is not in the document]
Comment (zeroasterisk) [reply tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md]: Also - @ryanmcginnis is correct - I'm about to update my edit to be more clear

**Pull request merged**